CMake Warning at CMakeLists.txt:18 (find_package):
  By not providing "FindJetsonGPIO.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "JetsonGPIO", but CMake did not find one.

  Could not find a package configuration file provided by "JetsonGPIO" with
  any of the following names:

    JetsonGPIOConfig.cmake
    jetsongpio-config.cmake

  Add the installation prefix of "JetsonGPIO" to CMAKE_PREFIX_PATH or set
  "JetsonGPIO_DIR" to a directory containing one of the above files.  If
  "JetsonGPIO" provides a separate development package or SDK, be sure it has
  been installed.


CMake Error at /opt/ros/foxy/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:60 (message):
  execute_process(/home/nvidia/anaconda3/bin/python3 -m rosidl_adapter
  --package-name arm_controller --arguments-file
  /home/nvidia/robotics-orin/build/arm_controller/rosidl_adapter__arguments__arm_controller.json
  --output-dir
  /home/nvidia/robotics-orin/build/arm_controller/rosidl_adapter/arm_controller
  --output-file
  /home/nvidia/robotics-orin/build/arm_controller/rosidl_adapter/arm_controller.idls)
  returned error code 1:

  Traceback (most recent call last):

    File "<frozen runpy>", line 198, in _run_module_as_main
    File "<frozen runpy>", line 88, in _run_code
    File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/__main__.py", line 19, in <module>
      sys.exit(main())
               ^^^^^^
    File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/main.py", line 53, in main
      abs_idl_file = convert_to_idl(
                     ^^^^^^^^^^^^^^^
    File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/__init__.py", line 18, in convert_to_idl
      from rosidl_adapter.msg import convert_msg_to_idl
    File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/msg/__init__.py", line 16, in <module>
      from rosidl_adapter.resource import expand_template
    File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/resource/__init__.py", line 19, in <module>
      import em

  ModuleNotFoundError: No module named 'em'

Call Stack (most recent call first):
  /opt/ros/foxy/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
  CMakeLists.txt:20 (rosidl_generate_interfaces)


